cpp/src/gandiva/tests/in_expr_test.cc

```diff
@@ -161,17 +161,6 @@ TEST_F(TestIn, TestInDouble) {
 
   // prepare input record batch
   auto in_batch = arrow::RecordBatch::Make(schema, num_records, {array0, array1});
-
```

Comment:
Why are you removing the checks below? Is this an error?

Reply:
I still don't understand why you removed these checks. Could you explain  or revert?